src/dataflow.rs

```diff
@@ -101,14 +107,15 @@ impl<S: Scope> FormatText<S> for Stream<S, String> {
                         && current_screen_dimension.is_some()
                         && current_font_size.is_some()
                     {
-                        let font_size = current_font_size.unwrap();
+                        let font_size = current_font_size.unwrap(); // font size
+                        let canvas_height = font_size * current_font_canvas_vstretch.unwrap_or(1.5); // font canvas height, factor controls vertical padding
```

Comment:
Instead of providing a default value, I think it's better to not perform any action if this parameter is not supplied. This will make maintenance simpler. Instead, add the `current_font_canvas_vstretch.is_some()` to the `if` before this block, i.e. on line 108.